### ISSUE TITLE
git_prompt_list_themes does not show colors

### ISSUE THREAD

**Issue by tessus** [magicmonty/bash-git-prompt#258]
If you run `git_prompt_list_themes`, the output is without colors until you have changed into a git repo directory once.

**Comment by ogr3** [comment]
Can you be more specific on when this occurs?

I get the current theme in red and Custom in blue regardless of where I stand in the file system.

**Comment by tessus** [comment]
Not sure, how I can be more specific.

It always occurs unless I am in the directory `~/.bash-git-prompt`. But when I add this one line, it works everywhere.

**Comment by ogr3** [comment]
what version of bash? I can't reproduce in either cygwin nor os x.

**Comment by tessus** [comment]
Mac OSX 10.11.5

```
$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)
Copyright (C) 2007 Free Software Foundation, Inc.
```

**Comment by tessus** [comment]
Just tried with `version 4.3.42(1)-release`. Same problem.

But I think I knopw why you can't reproduce it. Open a new iTerm or a new Terminal window. Run `git_prompt_list_themes`. Do not change into a git repo directory first.

I have to revise my problem description:

**If you run `git_prompt_list_themes`, the output is without colors until you have changed into a git repo directory once.**

**Comment by ogr3** [comment]
Do you have `GIT_PROMPT_ONLY_IN_REPO=1` set in your environment?

That would explain the behavior, since the color definitions are loaded when the prompt is updated (in `git_prompt_config`).

I did a proposed fix for this in #260. That way the color loading is the same regardless of the prompt has ben initialized or not.

**Comment by tessus** [comment]
Yes, I have `GIT_PROMPT_ONLY_IN_REPO=1` set.

My fix #259 is shorter, yours #260 is more elegant. I'll close my pull request.
